docs/realm.js

```diff
@@ -119,6 +119,15 @@ class Realm {
      */
     static openAsync(config, callback, progressCallback) {}
 
+    /**
+     * Return a configuration for a default synced Realm. The server URL for the current user will be used as base for
+     * the URL for the synced Realm.
+     * @throws {Error} if zero or multiple users are logged in
+     * @returns {Realm~Configuration} - a configuration matching a default synced Realm.
+     * @since 2.3.0
+     */
+    static defaultSyncConfiguration() {}
```

Comment:
It looks like you renamed this in the documentation, but not in the change log or implementation.

Reply:
Damn, forgot to push